src/main/java/com/salesforce/dev/pages/Leads/LeadViewDetail.java

```diff
@@ -0,0 +1,40 @@
+package com.salesforce.dev.pages.Leads;
+
+import com.salesforce.dev.framework.DriverManager;
+import com.salesforce.dev.pages.Base.ViewDetailBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by Ariel Mattos on 06/09/2015.
+ */
+public class LeadViewDetail extends ViewDetailBase {
+    public LeadViewDetail(WebDriver driver) {
+        super.driver = driver;
+        super.wait = DriverManager.getInstance().getWait();
+        PageFactory.initElements(driver, this);
+        try {
+            wait.withTimeout(10, TimeUnit.SECONDS)
+                    .until(ExpectedConditions.visibilityOf(viewSelected));
+        } catch (WebDriverException e) {
+            throw new WebDriverException(e);
+        } finally {
+            wait.withTimeout(15, TimeUnit.SECONDS);
```

Comment:
Magic number http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad
Try to use a constant for the timeout.
Review the constants in the project.